src/styles/components/card.scss

```diff
@@ -0,0 +1,108 @@
+@import "../core/settings";
+@import "../core/mixins";
+@import "../core/functions";
+/*!
+    card structure
+    .tn-card+(--static)+(.is-disabled)
+        .tn-card__media+(--fill)
+        .tn-card__content
+            .tn-card__header, .tn-card__description, .tn-card__label
+*/
+//set component name using namespace function, then use #{$block} below
+$block: ns(card);
```

Comment:
added comment here and changed the name spacing a bit. see `core/_settings` where namespace is set for the library, then see `core/_functions` for the `ns()` code. For the components, the line here is part of the template where you only have to put the component name here and it will be output as `tn-card`, for example (#683 will automate this). If you don't pass an arg to the function, it will return only `tn-` which is what you see in the helpers. I added some comments to those as well.